doc/quickstart/configure.rst

```diff
@@ -676,13 +676,141 @@ The following project-specific options are available:
+and tailor it to your needs.
+
+To list the available example configuration files, run the command:
+
+.. code-block:: bash
+
+  esmvaltool config list
+
+To use one of the example configuration files, copy it to
+your configuration directory by running the command:
+
+.. code-block:: bash
+
+  esmvaltool config copy data-intake-esgf.yml
+
+where ``data-intake-esgf.yml`` needs to be replaced by the name of the example
+configuration you would like to use. The format of the configuration file
+is described in :mod:`esmvalcore.io`.
+
+There are three modules available as part of ESMValCore that provide data sources:
+
+- :mod:`esmvalcore.io.intake_esgf`: Use the
+  `intake-esgf <https://intake-esgf.readthedocs.io>`_ library to load data that
+  is available from ESGF.
+- :mod:`esmvalcore.local`: Use glob patterns to find files on a filesystem.
+- :mod:`esmvalcore.esgf`: Use the legacy `esgf-pyclient
+  <https://esgf-pyclient.readthedocs.io>`_ library to find and download data
+  from ESGF.
+
+Adding a custom data source is relatively easy and is explained in
+:mod:`esmvalcore.io.protocol`.
+
+There are various use cases and we provide example configurations for each of
+them below.
+
+Personal computer
+`````````````````
+
+On a personal computer, the recommended setup can be obtained by running the
+commands:
+
+.. code-block:: bash
+
+    esmvaltool config copy data-intake-esgf.yml
+    esmvaltool config copy data-local-esmvaltool.yml
+
+This will use the :mod:`esmvalcore.io.intake_esgf` module to access data
+that is available through ESGF and use :mod:`esmvalcore.local` to find
+observational and reanalysis datasets that have been
+:ref:`CMORized with ESMValTool <esmvaltool:inputdata_observations>`
+(``OBS6`` and ``OBS`` projects for CMIP6- and CMIP5-style CMORization
+respectively) or are supported in their :ref:`native format <read_native_datasets>`
+through the ``native6`` project.
+
+.. warning::
+
+    It is important to configure intake-esgf for your system as described in
+    :mod:`esmvalcore.io.intake_esgf` before running the tool with this
+    configuration.
+
+HPC system
+``````````
+
+On HPC systems, data is often stored in large shared filesystems. We have
+several example configurations for popular HPC systems. To list the available
+example files, run the command:
+
+.. code-block:: bash
+
+  esmvaltool config list data-hpc
+
+If you are using one of the supported HPC systems, for example Jasmin, you can
+copy the example configuration file by running the command:
+
+.. code-block:: bash
+
+  esmvaltool config copy data-hpc-badc.yml
+
+and you should be good to go. If your HPC system is not supported yet, you can
+copy one of the other example configuration files, e.g. ``data-hpc-dkrz.yml``
+and tailor it for your system.
+
+.. warning::
+
+    It is important to configure intake-esgf for your system as described in
+    :mod:`esmvalcore.io.intake_esgf` before running the tool with this
+    configuration.
```

Comment:
here even more important to warn about setting the correct dll dir for intake, and have correct permissions on it